boards/microchip/pic32cxsg41_cult/pic32cxsg41_cult_defconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 Microchip
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_MICROCHIP_PIC32CXSG_XOSC32K=y
+CONFIG_SOC_MICROCHIP_PIC32CXSG_XOSC32K_AS_MAIN=y
```

Comment:
use devicetree

Reply:
I mean there is, Kconfig is for software configuration, the clock crystal source selection is not software, it goes in Kconfig

Reply:
Is this a block?

Reply:
it is, clock is hardware and hardware is described in devicetree, just as linux does it